config/settings/email.py

```diff
@@ -0,0 +1,12 @@
+from config.env import env
+from open_schools_platform.utils.email_utils import VKEmailService
+
+transport = env("EMAIL_SERVICE_TRANSPORT")
+
+if transport == "VK":
+    email_transport = VKEmailService
+
+
+class SendEmailService:
+    def __init__(self):
+        self.email_transport = email_transport
```

Comment:
Please, locate it to another place. For example, email_utils
setting should not contain any classes or functions